src/panels/AddSecretPanel/AddSecretPanel.test.tsx

```diff
@@ -0,0 +1,12 @@
+import { screen } from "@testing-library/react";
+
+import { renderComponent } from "testing/utils";
+
+import AddSecretPanel, { TestId } from "./AddSecretPanel";
+
+describe("AddSecretPanel", () => {
+  it("renders", async () => {
```

Comment:
This is just a placeholder test for when the functionality is added in https://warthogs.atlassian.net/browse/WD-8546.